packages/core/src/observables/track.ts

```diff
@@ -75,13 +79,16 @@ export function trackParticipantPairsObservable(
   room: Room,
   sources: Track.Source[],
   options: TrackParticipantPairsObservableOptions,
-): Observable<TrackParticipantPair[]> {
+): Observable<{ trackBundles: TrackParticipantPair[]; participants: Participant[] }> {
```

Comment:
I assume this is only temporary that you return them separately? I would have imagined all of them to be part of trackBundle now?

Reply:
No, this is intentional. For the placeholder thing to work, we need all participants in the room, not just those with tracks/publications. This is also the reason why all participants are returned separately.

Reply:
If the return type is a `bundle` in the format of `{participant: Participant, pub: Publication | undefined, track: Track | undefined }` it seems like we could get away with the bundle as the single return type and treat objects that do not have a publication as a placeholder?